### ISSUE TITLE
Request - add same site cookie attribute optional

### ISSUE THREAD

**Issue by yoyoprs** [vimalloc/flask-jwt-extended#34]
According to the security benefit cited here : https://www.sjoerdlangkemper.nl/2016/04/14/preventing-csrf-with-samesite-cookie-attribute/

Is it possible to include this in global options and on cookies ?

**Comment by vimalloc** [comment]
I haven't seen samesite cookies before, that is very interesting. Thanks for the read!

I would prefer to add this functionality via a flask/werkzeug helper method instead of manually modifying cookies myself, to keep things more secure and maintainable. I'll track this being integrated [here](https://github.com/pallets/flask/issues/2030) and [here](https://github.com/pallets/werkzeug/pull/1046). Once it it has been added, I will include it as an option in this library.

Cheers!

**Comment by yoyoprs** [comment]
Perfect ! Another question because we are talking about cookies.
Why don't you clean the CSRF Token with _unset_jwt_cookies_ method ?

**Comment by vimalloc** [comment]
Good catch. That is entirely an oversite on my end. I'll get a new version of this pushed out which unsets those as well. 👍
